dataactvalidator/validation_handlers/validationManager.py

```diff
@@ -522,6 +526,12 @@ def runCrossValidation(self, jobId, interfaces):
         errorDb.writeAllRowErrors(jobId)
         interfaces.jobDb.markJobStatus(jobId, "finished")
         CloudLogger.logError("VALIDATOR_INFO: ", "Completed runCrossValidation on submissionID: "+str(submissionId), "")
+        # Update error info for submission
+        interfaces.jobDb.getSubmissionErrorInfo(submissionId)
+        # TODO: Remove temporary step below
+        # Temporarily set publishable flag at end of cross file, remove this once users are able to mark their submissions
+        # as publishable
+        interfaces.jobDb.setPublishableFlag(submissionId, True)
```

Comment:
It's possible (probable, in fact) that I'm misunderstanding what `pulblishableFlag` is supposed to represent. Because of its name, I took it to mean that a submission is publishable if it passed all the validations. But if I read this correctly, we're setting publishable to True regardless of validation outcome.

If this is the intended behavior (i.e., the data store is also going to check the # of errors to determine the "good" submissions), does it make sense to use a different name? validation_complete?